packages/widgetbook_cli/bin/helpers/exceptions.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:widgetbook_exception/widgetbook_exception.dart';
+
+class WidgetbookDeployException implements WidgetbookException {
+  WidgetbookDeployException({String? message})
+      : _message =
+            message ??= 'An error occurred while deploying your Widgetbook to '
+                'Widgetbook Cloud. Please try again';
+
+  final String? _message;
+
+  @override
+  String get message => _message!;
+}
+
```

Comment:
In general the error messages are quite inconsistent in style for this file. Error messages should all end with a `.`. 

E.g. `An error occurred while deploying your Widgetbook to '
                'Widgetbook Cloud. Please try again` should be `An error occurred while deploying your Widgetbook to '
                'Widgetbook Cloud. Please try again.`